### DISCUSSION TITLE
visionOS support

### DISCUSSION THREAD

**Discussion by mkrus** [KhronosGroup/MoltenVK#2138] [Q&A]
Can anyone provide insights as to the visionOS support for moltenVK? I noticed so comments about it being supported for v1.2.5 but the matching LunarG does not include the binary package and I could not get the source to build for that tag or current HEAD of main.
Any clues?

**Comment by billhollings** [comment]
VisionOS is nominally supported, in that MoltenVK builds for the platform.

However, testing is very limited, and we have not made many changes regarding format, or functionality that might be different between a VisionOS platform and, say, iOS.

**Reply by mkrus** [reply]
thanks. As far as I can tell though, it doesn't build for me...
In `MVKSwapchain.mm:193: auto* screen = getCAMetalLayer().screenMVK`
screenMVK is not defined for visionOS
